src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -2325,6 +2328,61 @@ void canFulfilPageAndHoldRequestsWithMixedLevels(String pageRequestLevel, String
     assertThat(fetchRequestJson(holdRequest), isClosedFilled());
   }
 
+  @Test
+  void canCheckoutItemWhenTitleLevelPageRequestsExistForDifferentItemsOfSameInstance() {
+    configurationsFixture.enableTlrFeature();
+
+    List<ItemResource> items = itemsFixture.createMultipleItemsForTheSameInstance(4);
+    UUID instanceId = items.iterator().next().getInstanceId();
+
+    List<String> pagedItemIds = Stream.of(
+        requestsFixture.placeTitleLevelPageRequest(instanceId, usersFixture.steve()),
+        requestsFixture.placeTitleLevelPageRequest(instanceId, usersFixture.jessica()),
+        requestsFixture.placeTitleLevelPageRequest(instanceId, usersFixture.charlotte()))
+      .map(IndividualResource::getJson)
+      .map(json -> json.getString("itemId"))
+      .collect(Collectors.toList());
+
+    ItemResource nonPagedItem = items.stream()
+      .filter(not(item -> pagedItemIds.contains(item.getId().toString())))
+      .findFirst()
+      .orElseThrow(() -> new AssertionError("Failed to find non-paged item"));
+
+    checkOutFixture.checkOutByBarcode(nonPagedItem, usersFixture.james());
+  }
+
+  @Test
+  void cannotCheckoutItemWhenTitleLevelPageRequestExistsForSameItem() {
```

Comment:
Is the logic that is being checked by this test any different from cannotCheckoutWhenItemIsRequestedByTitleLevelRequestOfDifferentUser?

Reply:
It is. I removed the test you mentioned.